e2e/client/playwright/login.spec.ts

```diff
@@ -0,0 +1,54 @@
+import {test, expect} from '@playwright/test';
+import {s, restoreDatabaseSnapshot} from './utils';
+
+// Reset storage state for this file to avoid being authenticated
+test.use({storageState: {cookies: [], origins: []}});
+
+test.describe('login', () => {
+    test.beforeEach(async () => {
+        await restoreDatabaseSnapshot();
+    });
+
+    test('form renders modal on load', async ({page}) => {
+        await page.goto('/');
+
+        await expect(page.locator(s('login-page', 'submit'))).toBeVisible();
+    });
+
+    test('user can log in and logout', async ({page}) => {
+        await page.goto('/');
+
+        await page.locator(s('login-page', 'username')).fill('admin');
+        await page.locator(s('login-page', 'password')).fill('admin');
+        await page.locator(s('login-page', 'submit')).click();
+
+        // Wait for navigation to workspace
+        await expect(page.locator(s('dashboard'))).toBeVisible();
+        await expect(page).toHaveURL(/.*\/#\/workspace/);
+
+        // Check current user
+        await page.locator(s('my-profile')).click();
+        await expect(page.locator('.current-user__username')).toHaveText('admin');
```

Comment:
would be better to access this by `data-test-id` (and add it if not present yet)